src/web/templates/admin/accounts/account_modal.html

```diff
@@ -11,7 +11,7 @@ <h2 class="modal-title">
                     {% if admin_account.anonymous %}
                         {{ _('Editing of unauthenticated access') }}
                     {% else %}
-                        {{ _('Editing of account [%(username)s]', username=admin_account.username) }}
+                        {{ _('Editing of account [%(account_name)s]', account_name=admin_account.full_name) }}
```

Comment:
The titles were modified on every modal before the roles merge. Please also edit the other titles (Create account, Edit anonymous access)
```suggestion
                        {{ _('Edit account') }}
```